src/pages_/_document.tsx

```diff
@@ -32,9 +32,8 @@ export default class MyDocument extends Document {
     const { styleTags } = this.props;
 
     return (
-      <html lang="en">
+      <Html lang="en">
```

Comment:
Just a quick note: I think we should handle this dynamically based on the locale rather than using a static value.